Dockerfile

```diff
@@ -154,4 +210,17 @@ COPY moodle-shib.conf /etc/apache2/conf-enabled/
 COPY docker-entrypoint.d/* /docker-entrypoint.d/
 COPY 000-default.conf /etc/apache2/sites-available/
 
+# add modified files for Language Icon
+COPY themes/maker-header.mustache /var/www/html/theme/maker/templates/header.mustache
+COPY themes/boost-language_menu.mustache /var/www/html/theme/boost/templates/language_menu.mustache
+COPY themes/updated-icon-designs-4.1/mod/ /var/www/html/theme/maker/pix_plugins/mod
+
+COPY themes/updated-icon-designs-4.1/core/f/ /var/www/html/pix/f/
+
+##COPY themes/lib/outputrenderers.php /var/www/html/lib/
+RUN sed -i.bak '/public function lang_menu/ i\ public function get_language() {return current_language();}\n' /var/www/html/lib/outputrenderers.php
+
+RUN sleep 3 && echo "\$THEME->removedprimarynavitems = ['courses'];" >> /var/www/html/theme/maker/config.php
+##RUN sleep 3 && sed -i "\$a\\n\\n\$THEME->removedprimarynavitems = ['courses'];" /var/www/html/theme/maker/config.php
```

Comment:
Is this comment still needed?

Reply:
Yes, for line 223—this is a requirement from Kate to ensure that the course is not displayed in the students' menu.